src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/SingleProjectTest.cs

```diff
@@ -0,0 +1,54 @@
+using System.IO;
+using System.Xml.Linq;
+using NUnit.Framework;
+using Xamarin.Android.Tasks;
+using Xamarin.Android.Tools;
+using Xamarin.ProjectTools;
+
+namespace Xamarin.Android.Build.Tests
+{
+	[Category ("Node-1")]
+	[Parallelizable (ParallelScope.Children)]
+	public partial class SingleProjectTest : BaseTest
+	{
+		[Test]
+		public void AndroidManifestProperties ()
+		{
+			var packageName = "com.xamarin.singleproject";
+			var applicationLabel = "My Sweet App";
+			var versionName = "2.1";
+			var versionCode = "42";
+			var proj = new XamarinAndroidApplicationProject ();
+			proj.AndroidManifest = proj.AndroidManifest
```

Comment:
Likewise, if the default `dotnet new android` template no longer contains an `AndroidManifest.xml` -- and why should it now? -- then these `.Replace()` calls don't make sense.  Instead, `project.AndroidManifest` should be `null` (or some other value to prevent it from being created).

Reply:
Even in the case of "legacy" Xamarin.Android, if we're setting `$(GenerateApplicationManifest)`=True, then we shouldn't need `AndroidManifest.xml` at all, right?

Reply:
If anyone comes across this, we haven't completely removed `AndroidManifest.xml` yet.

The big feature missing to do this is app icons.